src/pres/Navigator.js

```diff
@@ -8,30 +8,40 @@ import * as Constants from '../app/Constants'
 import {trackEvent} from '../app/Analytics'
 
 export default class Navigator extends React.Component {
-    
+
     constructor(props){
         super(props)
         this.openingManager = new OpeningManager(this.props.variant)
         this.state = {
             currentMove:0,
-          }      
-          window.addEventListener("keydown",this.keyHandler.bind(this))
-  
+        }
+        window.addEventListener("keydown",this.keyHandler.bind(this))
     }
     keyHandler(e){
         switch(e.keyCode) {
-          case 37:
-            this.previous(e, "keyboard")
-          break
-          case 39:
-            this.next(e, "keyboard")
-            break
-          default:
-            break
+            case 37:
+                this.previous(e, "keyboard");
+                break
+            case 39:
+                this.next(e, "keyboard")
+                break
+            default:
+                break
+        }
+    }
+
+    scrollHandler(e) {
+        e.preventDefault();
```

Comment:
Could you add a sticky setting to disable this like you added for the dark mode? I personally do not like this and would like to keep it disabled.

Reply:
Would you like the default behaviour to be enabled or disabled?

Reply:
lets keep it disabled for now.